geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -99,3 +101,68 @@ def K_diag(self, X: B.Numeric, **parameters) -> B.Numeric:
         weights = self.eigenvalues(**parameters)  # [M, 1]
         Phi = self.eigenfunctions()
         return Phi.weighted_outerproduct_diag(weights, X, **parameters)  # [N,]
+
+
+class MaternIntegratedKernel(BaseGeometricKernel):
+    def __init__(
+        self,
+        space: Hyperbolic,
+        nu: float,
+        num_points_t: int,
+    ):
+
+        super().__init__(space)
+        self.nu = nu
+        self.num_points_t = num_points_t  # in code referred to as `T`.
+
+    def link_function(self, distance: B.Numeric, t: B.Numeric, lengthscale: B.Numeric):
+        """
+        This function links the heat kernel to the Matérn kernel, i.e., the Matérn kernel correspond to the integral of
+        this function from 0 to inf.
+        Parameters
+        ----------
+        :param distance: precomputed distance between the inputs
+        :param t: heat kernel lengthscale
+        Returns
```

Comment:
Parameter description for the lengthscale missing:
:param lenghtscale: lengthscale parameter of the kernel